### ISSUE TITLE
1.4.3 Contrast (Minimum) (AA)

### ISSUE THREAD

**Issue by renkesn** [UNLV-FIA/UNLV-FIA-Group#81]
### 1.4.3 Contrast (Minimum) - Level AA
The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: 

- **Large Text:** Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
- **Incidental:** Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
- **Logotypes:** Text that is part of a logo or brand name has no contrast requirement.

[Understanding 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)
[How to Meet 1.4.3](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=131%2C133%2C141%2C143%2C211%2C212#contrast-minimum)
[How to Test 1.4.3](https://usfs.box.com/s/8q6mf7c5xpn73zhfwdi0pgnijgfs4lvf)

**Trusted Tester ID:**
- [x] Test 13.C contrast

**Accessibility Principle:** Perceivable (Distinguishable)

**Comment by AlanaBayless** [comment]
All three pages pass 1.4.3 Contrast, all of the text has a contrast ratio of at least 4.5:1.
